### ISSUE TITLE
Custom link (instead of popup)

### ISSUE THREAD

**Issue by danikaze** [fmos/wp-photonav#6]
How about adding a link attribute to open a new page when clicking in the panorama?
As it's a div element, you can't do <a ...><div ...></div></a> so it would be a good idea to add something like this [photonav ... link='http://foo.bar' ]

**Comment by fmos** [comment]
This could interfere with the ColorBox feature. What behaviour would you expect, when ColorBox is enabled? Probably the link should replace the ColorBox function?! Have you thought about this?
 
-----Ursprüngliche Nachricht-----
Von:danikaze notifications@github.com
Gesendet:Fr 07.06.2013 07:54
Betreff:[wp-photonav] Link attr? (#6)
An:fmos/wp-photonav wp-photonav@noreply.github.com; 

How about adding a link attribute to open a new page when clicking in the panorama?
 As it's a div element, you can't do

  so it would be a good idea to add something like this [photonav ... link='http://foo.bar' ]

**Comment by danikaze** [comment]
Yeah I know, but there's shouldn't be a problem to give to the user the option of using a link or a color box. I personally don't use ColorBox so... it would be useful to get a link option ;)

**Comment by fmos** [comment]
Maybe it would make sense to introduce an option for the "popup" parameter, something like "popup=link:http://..."

**Comment by danikaze** [comment]
Well, I don't use a popup so it wouldn't be useful for me .P

**Comment by fmos** [comment]
Yes, but then it should be obvious that "colorbox" and "link" cannot be used at the same time. Of course the link would not really open a popup.
